assets/xml/objects/object_ru1.xml

```diff
@@ -1,69 +1,80 @@
+        <Animation Name="gRutoChildAnim_009060" Offset="0x9060"/>
+        <Animation Name="gRutoChildWait2Anim" Offset="0x97B8"/>
+        <DList Name="gRutoChildTorsoDL" Offset="0xC0A0"/>
+        <DList Name="gRutoChildLeftThighDL" Offset="0xC188"/>
+        <DList Name="gRutoChildLeftShinDL" Offset="0xC3D8"/>
+        <DList Name="gRutoChildLeftFootDL" Offset="0xC630"/>
+        <DList Name="gRutoChildRightThighDL" Offset="0xC7D0"/>
+        <DList Name="gRutoChildRightShinDL" Offset="0xCA20"/>
+        <DList Name="gRutoChildRightFootDL" Offset="0xCC78"/>
+        <DList Name="gRutoChildChestDL" Offset="0xCE18"/>
+        <DList Name="gRutoChildRightUpperArmDL" Offset="0xD0B8"/>
+        <DList Name="gRutoChildRightFinDL" Offset="0xD308"/>
+        <DList Name="gRutoChildDLRightHand" Offset="0xD5E0"/>
+        <DList Name="gRutoChildLeftUpperArmDL" Offset="0xD870"/>
+        <DList Name="gRutoChildLeftFinDL" Offset="0xDAC0"/>
+        <DList Name="gRutoChildLeftHandDL" Offset="0xDDA8"/>
+        <Texture Name="gRutoChildEyeTLUT" OutName="eye_tlut" Format="rgb5a1" Width="16" Height="16" Offset="0xE038"/>
+        <Texture Name="gRutoChildMouthTLUT" OutName="mouth_tlut" Format="rgb5a1" Width="16" Height="12" Offset="0xE238"/>
+        
+        <Texture Name="gRutoChildEyeOpenTex" OutName="eye_open" Format="ci8" Width="32" Height="32" Offset="0xE3B8"/>
+        <Texture Name="gRutoChildEyeHalfTex" OutName="eye_half" Format="ci8" Width="32" Height="32" Offset="0xF238"/>
+        <Texture Name="gRutoChildEyeClosedTex" OutName="eye_closed" Format="ci8" Width="32" Height="32" Offset="0xF638"/>
+        <Texture Name="gRutoChildEyeRollLeftTex" OutName="eye_roll_left" Format="ci8" Width="32" Height="32" Offset="0xFE38"/>
+        <Texture Name="gRutoChildEyeHalf2Tex" OutName="eye_half_2" Format="ci8" Width="32" Height="32" Offset="0x10238"/>
+        <Texture Name="gRutoChildEyeHalfWithBlushTex" OutName="eye_half_with_blush" Format="ci8" Width="32" Height="32" Offset="0x10A38"/>
+
+        <Texture Name="gRutoChildBlueSkinGradientTex" OutName="blue_skin_gradient" Format="rgb5a1" Width="8" Height="8" Offset="0xE7B8"/>
+
+        <Texture Name="gRutoChildMouthClosedTex" OutName="mouth_closed" Format="ci8" Width="32" Height="32" Offset="0xE838"/>
+        <Texture Name="gRutoChildMouthFrownTex" OutName="mouth_frown" Format="ci8" Width="32" Height="32" Offset="0xFA38"/>
+        <Texture Name="gRutoChildMouthOpenTex" OutName="mouth_open" Format="ci8" Width="32" Height="32" Offset="0x10638"/>
+
+        <Texture Name="gRutoChildEarTex" OutName="ear" Format="rgb5a1" Width="16" Height="16" Offset="0xEC38"/>
+        <Texture Name="gRutoChildBlueSkinPatchesTex" OutName="blue_skin_patches" Format="rgb5a1" Width="16" Height="16" Offset="0xEE38"/>
+        <Texture Name="gRutoChildUnkHeadCircle" OutName="tex_0000F038" Format="rgb5a1" Width="16" Height="16" Offset="0xF038"/> <!-- TODO rename me-->
```

Comment:
Tex

Reply:
looks like you had the same issues in multiple files. i pointed out the one in ru1 and you fixed it in ma1 lol